script/sync_prod_to_gcp/README.md

```diff
@@ -99,5 +99,25 @@ OTOH, because of this, it is rather important for Stanza to be running.
 Currently, the max file size is set to 4MiB, with 10 log files. It should be fine for a few days.
 
 
+# Submissions to GCP service
 
+## General
+
+A systemd service "submssions-to-gcp" subscribes to GCP pub/sub queue for published and 
+copy the submission tarball and abstract file to GCP bucket.
+
+
+## Service setup
+
+There is a (template) unit file. The service process runs one process per web node.
+
+    sudo systemctl enable submissions-to-gcp@sync-node.service
```

Comment:
I usually think of the @ as for systemd with multiple copies of a service. Why are there two copies? What is the difference between dync-node and sync-node? Please add that to this doc.

Reply:
Good point, the service file should not have an `@` in the filename, since we don't reuse this information as in `%i` somewhere in the systemd unit.